src/makeLottiePlayer.js

```diff
@@ -5,34 +5,35 @@ import React, {
 import equal from 'fast-deep-equal/es6/react';
 import clone from 'rfdc/default';
 
-import { propTypes, defaultProps } from './props';
+import propTypes from './propTypes';
 
 const makeLottiePlayer = ({ loadAnimation }) => {
-  const Lottie = memo(forwardRef(({
-    animationData,
-    path,
-
-    play,
-    speed,
-    direction,
-    segments: segmentsIn,
-    goTo,
-    useSubframes,
-
-    renderer,
-    loop,
-    rendererSettings: rendererSettingsIn,
-
-    audioFactory,
-
-    onLoad,
-    onComplete,
-    onLoopComplete,
-    onEnterFrame,
-    onSegmentStart,
-
-    ...props
-  }, forwardedRef) => {
+  const Lottie = memo(forwardRef((params, forwardedRef) => {
+    const {
+      animationData = null,
+      path = null,
+
+      play = null,
+      speed = 1,
+      direction = 1,
+      segments: segmentsIn = null,
+      goTo = null,
+      useSubframes = true,
+
+      renderer = 'svg',
+      loop = true,
+      rendererSettings: rendererSettingsIn = {},
```

Comment:
Wouldn't this cause renderSettings identity to change on every render, thereby causing excessive triggers of children/effects etc using it?